packages/components/src/VirtualizedList/CellActions/ActionsColumn.component.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Content from '../Content.component';
+import ActionsConfiguration from './ActionsColumn.configuration';
+
+function ActionsColumn(props) {
+	return <Content {...props} />;
```

Comment:
Why did you use a ColumnConfiguration file ?
Why didn't you directly define the cell renderer here ?

`return <Content {...props} cellRenderer={() => <CellActions {...props} />} />`

I think the all configuration file are very weird. Why have them if we have a dedicated component 🤔 ?
If it involves code duplication during the migration list, I don't see the problem. I prefer have something more readable at the end than avoiding duplication for a short period.

Reply:
Also now all columns juste implement RV.Column (behind Content), so this props is just the RV.Column props [api](https://github.com/bvaughn/react-virtualized/blob/master/docs/Column.md). 

To avoid confusion a documentation regrouping all the columns created here and the RV.Column api will be very helpfull I think.

Reply:
And one more thing, it could be nice to precise that with our current impl of RV you can't put code in the ***Column component (won't be executed).
I don't know if you have planned it in your pr series (IMHO its not mandatory for now, but nice for later)